docs/state.md

```diff
@@ -29,3 +29,13 @@ may prefer to just edit the config file with `kops edit cluster`.
 
 Because the configuration is merged, this is how you can just specify the changed arguments when
 reconfiguring your cluster - for example just `kops create cluster` after a dry-run.
+
+## Moving state between S3 buckets
```

Comment:
Do we even want to recommend this?  What caused the need for this?

Reply:
It's not something we should recommend but useful information for someone who may need to do this for whatever reason (housekeeping etc). I ran into a similar case where I wanted to move multiple test clusters into a single state bucket.

Reply:
I can add an "Advanced Users only" notice or something if that helps. In my specific case I needed to move the state from an ephemeral AWS account to a permanent AWS account after a cluster's intended lifespan unexpectedly increased (if that makes sense). I thought the cluster was only going to be used for experimenting with kops but it ended up getting "promoted"